src/routes/console/project-[project]/databases/database-[database]/collection-[collection]/document-[document]/attributes/relationship.svelte

```diff
@@ -0,0 +1,157 @@
+                    {label}
+                </Label>
+            </div>
+            {#if relatedList.length === 0}
+                <Button
+                    text
+                    noMargin
+                    on:click={() => {
+                        showInput = true;
+                    }}>
+                    <span class="icon-plus" aria-hidden="true" />
+                    <span class="text">Add item</span>
+                </Button>
+            {/if}
+        </div>
+        <ul class="u-flex-vertical u-gap-4 u-margin-block-start-4">
+            {#each relatedList as doc, i}
+                <li class="u-flex u-gap-16">
+                    <output class="input-text is-read-only">
+                        <div class="u-flex u-cross-baseline u-gap-12">
+                            <span
+                                class="u-flex-basis-140 u-flex-shrink-0 u-text-start u-trim-start u-x-small u-color-text-gray">
+                                {doc}
+                                <!-- {doc.label} -->
+                            </span>
+                            <span class="u-flex-1 u-trim-1">
+                                <!-- {#each doc.data as info}
+                                        {info}
+                                    {/each} -->
+                            </span>
+                        </div>
+                    </output>
+                    <Button text noMargin ariaLabel={`Delete item ${i}`}>
+                        <span class="icon-x" aria-hidden="true" />
+                    </Button>
+                </li>
+            {/each}
+            {#if showInput}
+                <li class="u-flex u-gap-16">
+                    <InputSelectSearch
+                        {id}
+                        label="Rel"
+                        showLabel={false}
+                        required
+                        bind:search
+                        bind:value={relatedList[relatedList.length]}
+                        options={documentList?.documents?.map((n) => ({
+                            value: n.$id,
+                            label: n.$id
+                        })) ?? []} />
+                    <Button
+                        text
+                        noMargin
+                        ariaLabel={`Hide input`}
+                        on:click={() => (showInput = false)}>
+                        <span class="icon-x" aria-hidden="true" />
+                    </Button>
+                </li>
+            {/if}
+        </ul>
+
+        {#if relatedList.length > 0 && !showInput}
+            <Button
+                text
+                noMargin
+                on:click={() => {
+                    showInput = true;
+                }}>
+                <span class="icon-plus" aria-hidden="true" />
+                <span class="text">Add item</span>
+            </Button>
+        {/if}
+    </div>
+{:else}
+    <InputSelectSearch
```

Comment:
The demo day showed the input including some attributes and the document ID. Are we still planning for that?